.wokeignore

```diff
@@ -0,0 +1,20 @@
+# The following files can be ignored when running Woke.
+
+rules-woke.yaml
+conf.py
+*.pu
+tox.ini
+Makefile
+.travis.yml
+.github/workflows/pull-request.yml

```

Comment:
```suggestion
.github/workflows/*.yml
```
I think we will add more code checked as actions later.

Reply:
Modified.